modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -1503,6 +1533,33 @@ Added 1 constraint.
 ======
 
 
+[discrete]
+[[constraints-create-a-relationship-property-type-constraint-union]]
+=== Create a relationship property type constraint with a union type 
+
+_This feature was introduced in Neo4j 5.11._
+
+A closed dynamic union allows a relationship property to maintain some type flexibility whilst preventing unexpected values from being stored.
+
+
+.+CREATE CONSTRAINT+
+======
+
+.Query
+[source, cypher]
+----
+CREATE CONSTRAINT part_of_tags
+FOR ()-[part:PART_OF]-() REQUIRE part.tags IS :: STRING | LIST<STRING NOT NULL>
+----
+
+.Result
+[queryresult]
+----
+Added 1 constraint.
```

Comment:
@mnd999 I think you should probably update the SHOW CONSTRAINTS example at the bottom of this file to also include your 2 new constraints (should have failed docs tests?)

Reply:
That result seems hard coded. I did not realise that is supposed to represent all the constraints created in the rest of the document - although I guess as it's hard coded maybe it doesn't .

Reply:
it should, but yes it's hard coded - we don't generate docs from tests anymore so the result needs to be hard coded

I think the docs testing structure don't catch the 'error' of getting back more than we assert on, just if we don't get back things we assert on 🤷